web/third/bootstrap/css/bootstrap.css

```diff
@@ -464,7 +464,6 @@ label {
 }
 input[type="text"],
 input[type="password"],
-input[type="date"],
```

Comment:
We apparently have some dark theme CSS for `input[type="date"]`, though I think it was from before custom profile field date fields used flatpickr.

Reply:
Yeah we have such CSS for a couple of other input elements as well. Will remove in a separate PR.